frontend/test/__support__/enzyme.js

```diff
@@ -155,3 +147,60 @@ export async function fillAndSubmitForm(wrapper, values) {
   await fillFormValues(wrapper, values);
   submitForm(wrapper);
 }
+
+function findByText(wrapper, text) {
+  return wrapper.find(`[children=${JSON.stringify(text)}]`);
+}
+function findByIcon(wrapper, icon) {
+  return wrapper.find(`.Icon-${icon}`);
+}
+
+// adds helper  methods to enzyme ReactWrapper
+addReactWrapperMethod("click", click);
+addReactWrapperMethod("clickButton", clickButton);
+addReactWrapperMethod("findByText", findByText);
+addReactWrapperMethod("findByIcon", findByIcon);
+addReactWrapperMethod("setInputValue", setInputValue);
+
+// creates the magic "async" namespace for `find` methods
+Object.defineProperty(ReactWrapper.prototype, "async", {
+  get: function() {
+    if (!this.__async) {
+      this.__async = {
+        find: asyncFind(this, "find"),
+        findWhere: asyncFind(this, "findWhere"),
+        findByText: asyncFind(this, "findByText"),
+        findByIcon: asyncFind(this, "findByIcon"),
+      };
+    }
+    return this.__async;
+  },
```

Comment:
Is this getter only for lazy instantiation or does this do something else?

Reply:
Yes, it's lazy instantiation.